test/unit/style/style.test.js

```diff
@@ -992,20 +992,19 @@ test('Style#addLayer', (t) => {
     });
 
     t.test('emits error on duplicates', (t) => {
+        t.stub(console, 'error');
```

Comment:
Is this necessary?

Reply:
hmm I guess not! left over from earlier debugging I think. removed!